ingest/src/updateEmbeddedContent.ts

```diff
@@ -76,19 +76,20 @@ export const updateEmbeddedContentForPage = async ({
 }): Promise<void> => {
   const contentChunks = await chunkPage(page, chunkOptions);
 
-  const embeddedContent = await Promise.all(
-    contentChunks.map(async (chunk): Promise<EmbeddedContent> => {
-      const { embedding } = await embed({
-        text: chunk.text,
-        userIp: "127.0.0.1",
-      });
-      return {
-        ...chunk,
-        embedding,
-        updated: new Date(),
-      };
-    })
-  );
+  const embeddedContent: EmbeddedContent[] = [];
+  // Process sequentially because we're likely to hit rate limits before any
+  // other performance bottleneck
+  for (const chunk of contentChunks) {
+    const { embedding } = await embed({
+      text: chunk.text,
+      userIp: "127.0.0.1",
```

Comment:
not important for this PR, but can you just get the IP of the system where the script is running?

Reply:
Possibly, though it would add a dependency on an external site? https://stackoverflow.com/questions/20273128/node-js-how-to-get-my-external-ip-address-in-node-js-app